activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/src/test/java/org/activiti/cmdendpoint/CommandEndpointKeycloakIT.java

```diff
@@ -48,9 +50,9 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @TestPropertySource("classpath:application-test.properties")
-@DirtiesContext
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_CLASS)
 @EnableBinding(MessageClientStream.class)
-public class CommandEndpointTest {
+public class CommandEndpointKeycloakIT extends KeycloakEnabledBaseTestIT {
```

Comment:
@ryandawsonuk I;m worried about having all the tests called KeycloakIT now... there is no need if we assume that keycloak will be there always right?